grammar.js

```diff
@@ -1010,6 +1005,16 @@ module.exports = grammar({
       ')',
     ),
 
+    group_concat: $ => seq(
+      field('name', $.keyword_group_concat),
+      '(',
+      optional($.keyword_distinct),
+      field('parameter', choice($._expression)),
+      optional($.order_by),
+      optional(seq($.keyword_separator, alias($._literal_string, $.literal))),
+      ')',
```

Comment:
in Postgres [this is possible with aggregate functions more generally](https://www.postgresql.org/docs/current/sql-expressions.html#SYNTAX-AGGREGATES)! e.g.

```
select count(distinct uid order by uid)
from table_a;
```

(although it doesn't really mean much with `count`)

I think putting this in `_aggregate_function` would be a good first step toward supporting that -- we can fill in the maxes mins etc and worry about extra stuff like the `filter` clause later.

Reply:
I've added an `_aggregate_expression` node to help generalize those cases: https://github.com/DerekStride/tree-sitter-sql/commit/61c6e23337ed72464dc5c614fc47f876c1ad6ad6